posthog/api/test/test_dashboard.py

```diff
@@ -659,6 +659,116 @@ def test_patch_api_as_form_data(self):
         self.assertEqual(response.status_code, 200, response.content)
         self.assertEqual(response.json()["name"], "replaced")
 
+    def test_can_hard_delete_insight_after_hard_deleting_dashboard(self) -> None:
```

Comment:
flyby comment. Why do we allow both hard and soft deletes? We should only do soft deletes really so people can undo (once we get around to implementing that)

Reply:
I guess they come for free with DRF and we've not removed them

Reply:
I don't think there's any point in having both and testing both. Let's only allow soft deletes and remove the ability to hard delete.